pkg/goldpinger/client.go

```diff
@@ -65,12 +65,14 @@ func PingAllPods(pods map[string]string) models.CheckResults {
 
 			CountCall("made", "ping")
 			timer := GetLabeledPeersCallsTimer("ping", hostIP, podIP)
+			start := time.Now()
 			resp, err := getClient(pickPodHostIP(podIP, hostIP)).Operations.Ping(nil)
 
 			channelResult.hostIPv4.UnmarshalText([]byte(hostIP))
 			var OK = (err == nil)
 			if OK {
-				channelResult.podResult = models.PodResult{HostIP: channelResult.hostIPv4, OK: &OK, Response: resp.Payload, StatusCode: 200}
+				responseTime := float64(time.Since(start).Nanoseconds()) / float64(int64(time.Millisecond))
```

Comment:
I'm wondering if it wouldn't be easier to just return nanoseconds ?
Or perhaps just do integer division, since we probably don't care about fractions of milliseconds ?

What do you think ?